package.json

```diff
@@ -65,14 +65,13 @@
     "lint": "eslint . --ext .ts",
     "release": "yarn && lerna publish --no-private --force-publish && yarn retype:updateversion",
     "release:manual": "node scripts/workspaceRun.mjs release",
+    "release:yalc": "(yarn entry:dist && yarn lerna exec npx yalc push); yarn entry:src",
```

Comment:
just to clarify, this doesn't actually release anything, just creates a build artifact that _could_ be released or mirrors what would be uploaded to npm?

Reply:
omg the level of wrapping here is terrifying: `yarn lerna exec npx yalc push`

Reply:
> just to clarify, this doesn't actually release anything, just creates a build artifact that could be released or mirrors what would be uploaded to npm?

Correct, it creates the build artifacts and publishes them to a local `yalc` registry

> omg the level of wrapping here is terrifying: yarn lerna exec npx yalc push

🙈